examples/hal/quad/main.rs

```diff
@@ -305,7 +305,7 @@ fn main() {
         pipeline
     };
 
-    println!("Pipeline: {:?}", pipeline);
+    //println!("Pipeline: {:?}", pipeline);
```

Comment:
nit: was this left commented out on purpose? We could just remove it if it's not useful

Reply:
it's really nit. The amount of stuff it prints on OSX (more than a page of text) is just to annoying. We can totally remove this for now.